### PR TITLE
Rename Administrators to AdministratorContacts

### PR DESCRIPTION
**Pull request by heaths**

### CODE REVIEW

File: sdk/keyvault/Azure.Security.KeyVault.Certificates/src/SubjectAlternativeNames.cs [hunk 1]
```diff
@@ -35,17 +35,17 @@ public SubjectAlternativeNames()
         /// <summary>
         /// Gets a collection of DNS names.
         /// </summary>
-        public ICollection<string> DnsNames => LazyInitializer.EnsureInitialized(ref _dnsNames);
+        public IList<string> DnsNames => LazyInitializer.EnsureInitialized(ref _dnsNames);
```
Comment (schaabs) [inline sdk/keyvault/Azure.Security.KeyVault.Certificates/src/SubjectAlternativeNames.cs]: I think there might be a race around here. According to the `LazyInitializer` docs [here](https://docs.microsoft.com/en-us/dotnet/api/system.threading.lazyinitializer.ensureinitialized?view=netframework-4.8#System_Threading_LazyInitializer_EnsureInitialized__1___0__)

>In the event that multiple threads access this method concurrently, multiple instances of T may be created, but only one will be stored into target. In such an occurrence, this method will not dispose of the objects that were not stored. If such objects must be disposed, it is up to the caller to determine if an object was not used and to then dispose of the object appropriately.

This leads me to believe that what is returned from the method might not be what's actually stored in `_dnsNames`.

Comment (heaths) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/src/SubjectAlternativeNames.cs]: You may be right. I was thinking of this default behavior from `Lazy<T>`:
> When multiple threads try to initialize a Lazy<T> instance simultaneously, all threads are allowed to run the initialization method (or the parameterless constructor, if there is no initialization method). The first thread to complete initialization sets the value of the Lazy<T> instance. That value is returned to any other threads that were simultaneously running the initialization method, unless the initialization method throws exceptions on those threads.
However, the implementation shows the documentation wrong (there's no way to dispose of created objects so you can't access them):
```csharp
try
{
    Interlocked.CompareExchange<T>(ref target, Activator.CreateInstance<T>(), default(T));
}
catch (MissingMethodException)
{
    throw new MissingMemberException(SR.Lazy_CreateValue_NoParameterlessCtorForT);
}
return target;
```
However (again), we should probably go by the docs.

Comment (heaths) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/src/SubjectAlternativeNames.cs]: BTW, that code is only executed if an initial volatile read of the `target` is null, so it's not like we're allocating a collection each time we check.

Comment (heaths) [reply sdk/keyvault/Azure.Security.KeyVault.Certificates/src/SubjectAlternativeNames.cs]: After talking offline, we'll leave as-is. I did file a doc bug against CoreFX @ dotnet/corefx#42643.

**Review by schaabs — APPROVED** [review]

**Pull request merged**